README.md

```diff
@@ -46,13 +46,13 @@ The above analogies illustrate several important tradeoffs between these systems
 
 ## 1.3 Security Considerations
 
-Each UCAN includes a constructive set of assertions of what it is allowed to do. Note that this is not a predicate: it is a positive assertion of rights. These are proven by providing a "witness" -- some cryptographically signed data showing that this is either owned by the UCAN issuer, or that it was delegated to them by the root owner.
+Each UCAN includes a constructive set of assertions of what it is allowed to do. Note that this is not a predicate: it is a positive assertion of rights. These are proven by providing a "witness" — some cryptographically signed data showing that this is either owned by the UCAN issuer, or that it was delegated to them by the root owner.
```

Comment:
The em-dash was intended

Reply:
The diff makes it a bit hard to tell, but this was replacing the double hyphens with an actual em dash character: 
![image](https://user-images.githubusercontent.com/285821/152260493-088c2642-15d6-417f-a88c-07bd255d3c76.png)

Happy to revert if you prefer the hyphens though!

Reply:
Ooh I see. In my markdown editor, this turns into an em-dash, but doesn't in GH. And then the preview heer doesn't look very em-dash-y. Okay makes sense; thanks!

Reply:
Yup, I was initially confused about your comment until I looked at things in the diff view!